acquia_lift.links.action.yml

```diff
@@ -0,0 +1,5 @@
+entity.acquia_lift_slot.add_form:
+  route_name: entity.acquia_lift_slot.add_form
```

Comment:
well, it's because acquia_lift_slot is the config entity identifier. otherwise you'll have to make it acquia_lift.entity.slot.add_form which makes even more sense. Going to change it to that.